scripts/rollup/forks.js

```diff
@@ -65,9 +65,26 @@ const forks = Object.freeze({
     return null;
   },
 
+  'react-reconciler/src/ReactFiberScheduler': (bundleType, entry) => {
+    if (entry === 'react-dom/unstable-new-scheduler') {
```

Comment:
Shouldn't the feature flag take care of this?

Reply:
It turns out the way I set up the ReactFiberScheduler wasn't sufficient to dead code eliminate the unused module exports. I initialized the exports to the old ones, then re-assign them to the new ones if the flag is on. But what I should do instead is initialize them to nothing, then conditionally assign once based on the flag.

This is an extra measure because I'm paranoid.

Reply:
I feel like this will just work to cover up mistakes that might have other downstream effects.

We can make sure that this is a noop in this PR.

For future PRs we should just refactor the old OR the new in each PR. That way the one that we're not changing should be a no-change.